care/utils/assetintegration/base.py

```diff
@@ -1,24 +1,29 @@
+import enum
 import json
 
 import requests
 from django.conf import settings
-from rest_framework.exceptions import APIException
+from django.core.cache import cache
+from rest_framework.exceptions import APIException, PermissionDenied
 
+from care.users.models import User
 from care.utils.jwks.token_generator import generate_jwt
 
 
 class BaseAssetIntegration:
     auth_header_type = "Care_Bearer "
 
+    class BaseAssetActions(enum.Enum):
+        UNLOCK_ASSET = "unlock_camera"
+        LOCK_ASSET = "lock_camera"
+        REQUEST_ACCESS = "request_access"
+
     def __init__(self, meta):
         self.meta = meta
         self.host = self.meta["local_ip_address"]
         self.middleware_hostname = self.meta["middleware_hostname"]
         self.insecure_connection = self.meta.get("insecure_connection", False)
 
-    def handle_action(self, action):
-        pass
-
```

Comment:
it would be good to have it there 
just update the signature and raise NotImplementedError